internal/model/model.go

```diff
@@ -1222,6 +1218,7 @@ nextSub:
 		TempNamer:     defTempNamer,
 		TempLifetime:  time.Duration(m.cfg.Options().KeepTemporariesH) * time.Hour,
 		CurrentFiler:  cFiler{m, folder},
+		MtimeRepo:     db.NewVirtualMtimeRepo(m.db, folderCfg.ID),
```

Comment:
It's always consulted, but only written to if an `os.Chtimes()` call fails. So as long as you're on a platform where that call doesn't fail, no record will be found.